File: frontend/src/types/community-posts-report/select-community-posts-report-dto.ts
import { BaseSelectCommunityReportDTO } from "../community-content-report/base-select-community-report-dto";
import { SelectReportedPostDTO } from "../community-content-report/select-reported-content-dto";

export interface SelectCommunityPostsReportDTO extends BaseSelectCommunityReportDTO {
    postId: string;
    post?: SelectReportedPostDTO | null;
}

File: platform-core-service/platform-core-service.Common/Models/DTOs/EntityDTO/CommunityPostsReport/SelectCommunityPostsReportDTO.cs
using platform_core_service.Common.Models.DTOs.EntityDTO.CommunityContentReport;

namespace platform_core_service.Common.Models.DTOs.EntityDTO.CommunityPostsReport
{
    public class SelectCommunityPostsReportDTO : BaseSelectCommunityReportDTO
    {
        public string PostId { get; set; } = null!;
        public SelectReportedPostDTO? Post { get; set; }
    }
}
